.github/scripts/visualize_stopping_patterns.py

```
@@ -0,0 +1,434 @@
+
+MAX_GROUPS = 20
+MAX_COMMENT_LENGTH = 65000
+
+SST_FILE = "data/5!station_station_types.csv"
+STATIONS_FILE = "data/3!stations.csv"
+TYPES_FILE = "data/4!types.csv"
+LINES_FILE = "data/2!lines.csv"
+
+
+def git_diff(base_ref: str, file_path: str) -> str:
+    result = subprocess.run(
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)